src/Hexagon.java

```diff
@@ -5,15 +5,19 @@
     private TerrainType terrainType;
     private int tileID;
     private int level;
-    private int numVillagersOnTop;
+    private int numberOfVillagersOnTop;
```

Comment:
Probably should use a boolean for numberOfVillagersOnTop because based on level we know that amount